docs/install.rst

```diff
@@ -7,14 +7,16 @@ Requirements
 
 Astropy has the following strict requirements:
 
-- `Python <http://www.python.org/>`_ 2.7, 3.4, 3.5 or 3.6
+- `Python <http://www.python.org/>`_ 3.5 or 3.6
 
   - Prior to Astropy v1.0, Python 3.1 and 3.2 were also supported.
 
   - Prior to Astropy v1.2, Python 2.6 was supported.
 
   - Prior to Astropy v2.0, Python 3.3 was supported.
 
+  - Prior to Astropy v3.0, Python 2.7 and 3.4 were supported.
```

Comment:
Maybe we should just delete this historical list, and keep only this new entry? If we keep all, I would suggest to reverse the order.

Reply:
I like the suggestion, details always can be found in the changelog.